components/images/fullpage-hero.css

```diff
@@ -22,7 +22,7 @@
 .ampstart-image-fullpage-hero .ampstart-image-credit {
   box-decoration-break: clone;
   background: var(--color-primary);
-  padding: 0 var(--space-2) 0.2rem;
```

Comment:
I think @wassgha - came up with this. There is no issue on this part of the template?